File: ios/DataToRN.m
#import <React/RCTBridgeModule.h>
#import <Foundation/Foundation.h>
#import "UIKit/UIKit.h"
@interface RCT_EXTERN_MODULE(RNShare, NSObject)

RCT_EXTERN_METHOD(share)
RCT_EXTERN_METHOD(someMethod:(RCTResponseSenderBlock)successCallback)

@end

File: ios/DataToRN.swift

import Foundation



@objc(RNShare)
class RNShare : NSObject {
    
  @objc static func requiresMainQueueSetup() -> Bool {
        return false
    }
  
  
  @objc func someMethod(_ successCallback: RCTResponseSenderBlock) -> Void {
    successCallback([NSNull(), HelloWorldWrapper().sayHello() as Any])
    
    }
  

}

